akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -95,15 +98,327 @@ Scala
+one which runs the data access layer and one for the number-crunching. Deployment of actors—for example by cluster-aware
+routers—can take node roles into account to achieve this distribution of responsibilities.
+
+The node roles are defined in the configuration property named `akka.cluster.roles`
+and typically defined in the start script as a system property or environment variable.
+
+The roles are part of the membership information in `MemberEvent` that you can subscribe to.
+
+## Failure Detector
+
+The nodes in the cluster monitor each other by sending heartbeats to detect if a node is
+unreachable from the rest of the cluster and has been downed and removed. Please see:
+
+* @ref:[Failure Detector specification](cluster-specification.md#failure-detector)
+* @ref:[Phi Accrual Failure Detector](failure-detector.md) implementation
+* [Using the Failure Detector](#using-the-failure-detector) 
+ 
+### Using the Failure Detector
+ 
+Cluster uses the `akka.remote.PhiAccrualFailureDetector` failure detector by default, or you can provide your by
+implementing the `akka.remote.FailureDetector` and configuring it:
+
+```
+akka.cluster.implementation-class = "com.example.CustomFailureDetector"
+```
+
+In the [Cluster Configuration](#configuration) you may want to adjust these
+depending on you environment:
+
+* When a *phi* value is considered to be a failure `akka.cluster.failure-detector.threshold`
+* Margin of error for sudden abnormalities `akka.cluster.failure-detector.acceptable-heartbeat-pause`  
+
+If you encounter suspicious false positives when the system is under load you should
+define a separate dispatcher for the cluster actors as described in [Cluster Dispatcher](#cluster-dispatcher).
+  
+## How to Test
+
+Akka comes with and uses several types of testing strategies:
+
+* @ref:[Testing](testing.md)
+* @ref:[Multi Node Testing](../multi-node-testing.md)
+* @ref:[Multi JVM Testing](../multi-jvm-testing.md)
+
+## Configuration
+
+There are several configuration properties for the cluster. Refer to the 
+@ref:[reference configuration](../general/configuration.md#config-akka-cluster) for full
+configuration descriptions, default values and options.
+
+### Cluster Info Logging
+
+You can silence the logging of cluster events at info level with configuration property:
+
+```
+akka.cluster.log-info = off
+```
+
+You can enable verbose logging of cluster events at info level, e.g. for temporary troubleshooting, with configuration property:
+
+```
+akka.cluster.log-info-verbose = on
+```
+
+### Cluster Dispatcher
+
+The cluster extension is implemented with actors. To protect them against
+disturbance from user actors they are by default run on the internal dispatcher configured
+under `akka.actor.internal-dispatcher`. The cluster actors can potentially be isolated even
+further onto their own dispatcher using the setting `akka.cluster.use-dispatcher`.
```

Comment:
Or made run on the same dispatcher to keep the number of threads down (not sure if worth mentioning though)

Reply:
Good point.